packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -848,12 +848,32 @@ function merge(template: any, part: any) {
     if (!dest) {
       template[section] = dest = src;
     } else {
-      // add all entities from source section to destination section
-      for (const id of Object.keys(src)) {
-        if (id in dest) {
-          throw new Error(`section '${section}' already contains '${id}'`);
+      if (typeof dest === 'object') {
+        // add all entities from source section to destination section
+        for (const id of Object.keys(src)) {
+          if (id in dest) {
+            throw new Error(`section '${section}' already contains '${id}'`);
+          }
+          dest[id] = src[id];
+        }
+      } else {
+        // the sections are strings, like Description and AWSTemplateFormatVersion
+        // treat each differently
+        switch (section) {
+          case 'Description':
+            // merge the two descriptions, with a newline in between
+            template[section] = `${dest}\n${src}`;
+            break;
```

Comment:
This is a weird choice. I would say either error or destination wins. 

In the case of CfnInclude, I think it's fair to say that if the description was altered after import, the altered description wins.

Reply:
This is not about "altering" - this is the case when `Description` was provided from multiple places. See the test for this:

```ts
    new CfnInclude(stack, 'T1', {
      template: {
        AWSTemplateFormatVersion: '2010-09-09',
        Description: 'Test 1',
      },
    });
    new CfnInclude(stack, 'T2', {
      template: {
        AWSTemplateFormatVersion: '2010-09-09',
        Description: 'Test 2',
      },
    });
```